weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundSettings/PlaygroundSettings.tsx

```diff
@@ -71,22 +77,24 @@ export const PlaygroundSettings: React.FC<PlaygroundSettingsProps> = ({
                 }
               />
 
-              <ResponseFormatEditor
-                responseFormat={playgroundState.responseFormat}
-                setResponseFormat={value =>
-                  setPlaygroundStateField(idx, 'responseFormat', value)
+              <StopSequenceEditor
+                stopSequences={playgroundState.stopSequences}
+                setStopSequences={value =>
+                  setPlaygroundStateField(idx, 'stopSequences', value)
                 }
               />
 
+              {/* TODO: N times to run is not supported for all models */}
+              {/* TODO: rerun if this is not supported in the backend */}
               <PlaygroundSlider
-                min={0}
-                max={2}
-                step={0.01}
+                min={1}
+                max={100}
+                step={1}
                 setValue={value =>
-                  setPlaygroundStateField(idx, 'temperature', value)
+                  setPlaygroundStateField(idx, 'nTimes', value)
                 }
-                label="Temperature"
-                value={playgroundState.temperature}
+                label="Completion iterations"
```

Comment:
Can we add a tooltip? I'm not the target audience but I don't really know what this means... Where are the multiple iterations going to end up?